src/components/generic/Modal/Modal.jsx

```diff
@@ -17,47 +14,34 @@ import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../../constants';
  * @returns {object}  JSX of the element
  */
 export default function Modal({
-  applicationId,
   children,
   className,
   onClose,
   title,
-  ...props
 }) {
-  const mainClasses = {
-    [`${WEBEX_COMPONENTS_CLASS_PREFIX}-modal`]: true,
-    [className]: !!className,
-  };
-  const testId = `${WEBEX_COMPONENTS_CLASS_PREFIX}-${Math.random().toString().slice(2)}`;
+  const cssClasses = webexComponentClasses('modal-content', className);
 
   return (
-    <MomentumModal
-      applicationId={applicationId}
-      className={classNames(mainClasses)}
-      htmlId={testId}
-      onHide={onClose}
-      show
-      // eslint-disable-next-line react/jsx-props-no-spreading
-      {...props}
-    >
-      <ModalHeader headerLabel={title} showCloseButton={!!onClose} />
-      <ModalBody>
-        {children}
-      </ModalBody>
-    </MomentumModal>
+    <div className="modal">
+      <div className={cssClasses}>
+        <div className="modal-header">
+          <h3 className="modal-title">{title}</h3>
```

Comment:
don't we have any modal without a title or without a close action? (eg: proceed without)